packages/css-framework/src/components/_switch.scss

```diff
@@ -1,7 +1,7 @@
 $border-radius: 2px;
 $background-color: color(neutral, 0);
 $border-color: color(neutral, 200);
-$active-background-color: color(neutral, white);
```

Comment:
while you are here... we talked about namespacing vars the other day. As it stands vars get put into a global scope so vars from one file can break another.

Reply:
Yeah this was me just modifying the pre-existing variables here.

I'm not entirely sold on the idea of having variables _within_ a particular component file. There's definitely times where using a variable in this situation is beneficial, however I've seen a number of instances where the variable is only used once. This means you'd actually be changing the value _outside_ of its context within the class definition. I think we should avoid this as much as possible, and I definitely agree with you about namespacing these variables where we use them.